README.md

```diff
@@ -41,38 +80,221 @@ Checkout [architectures notes](doc/Architecture.md)
+CREATE DATABASE ragnarok;
+GRANT ALL PRIVILEGES ON DATABASE ragnarok TO ragnarok;
+ALTER DATABASE ragnarok OWNER TO ragnarok;
+```
+
+After your database is up, you will need import the Game SQL (`/rust-ro/pg.sql`) to our new database. For that follow the steps:
+
+* Connect into the Docker `postgres_container` Bash
+* Log-in as user `postgres`
+* Run the `psql` command pointing which the import file to your new database
+
+```shell
+docker exec -it postgres_container bash
+su postgres
+psql ragnarok < pg.sql
+```
+
+With that you're ready to go to the next step. 
+
+#### 5.2.2 Setup DB: Locally
+
+If you have PostgreSQL installed in your machine, you will need to log-in into PSQL and create the user, dabase and give the necessary privilege for it:
+
+```shell
+sudo -u postgres psql
+```
+
+Run the queries below: 
+
+```sql 
+CREATE USER ragnarok WITH PASSWORD 'ragnarok';
+CREATE DATABASE ragnarok;
+GRANT ALL PRIVILEGES ON DATABASE ragnarok TO ragnarok;
+ALTER DATABASE ragnarok OWNER TO ragnarok;
+```
+
+After that, exit pgsql and import our `/rust-ro/db/pg.sql` via cli with:
+
+```shell
+ sudo -u postgres psql ragnarok < db/pg.sql
+```
+
+> If you're using the local version, don't forget to change the `database.port` number to `5433` in your `config.json` file. 
+
+
+### 5.3 Building the Binaries
+
+So far, we have a few executables being compiled together with the project:
+
+- **maps:** TODO description
+- **maps-tool:** TODO description
+- **packets:** TODO description
+- **packets-tool:** TODO description
+- **server:** TODO description
+- **skills-enum-generator:** TODO description
+
+To build, just go on the project root and run: 
+
+```shell
+cargo build
+```
+
+Simple like that! Now you're good to run your servers. 
+
+### 5.4 Running the Binaries
+
+After we have everyting set-up (binaries and database), we should run these binaries to make the game playable:
+
+- server
+- maps
+- packets
+
+
+#### 5.4.1 Running Servers
+
+To run the `server` binary, you will need a `ENV` variable called `DATABASE_PASSWORD` together with your command:
 
+```shell
+DATABASE_PASSWORD=ragnarok cargo run --bin=server
+```
 
+If everything goes right, you should receive something like this output:
+
+```
+exclude pattern sqlx
+class file -> _Global
+class file -> Test
+class file -> GlobalVariableTest
+class file -> Warper
+class file after filter -> _Global
+class file after filter -> Test
+class file after filter -> GlobalVariableTest
+class file after filter -> Warper
+INFO [server] load 39 scripts in 0.126 secs
+INFO [server::server::boot::warps_loader] load 2781 warps in 0.007 secs
+INFO [server::server::boot::mob_spawn_loader] load 3391 mob spawns in 0.022 secs
+INFO [server] load 897 map-cache in 0.05 secs
+INFO [server::proxy] Start proxy for Char proxy, 6123:6121
+INFO [server::proxy] Start proxy for map proxy, 6124:5121
+INFO [server::server] Server listen on 0.0.0.0:6901
+```
+
+#### 5.4.2 Running the Game
+
+TODO: roBrowser or what?
```

Comment:
as i support only a specific packetver for the moment i think robrowser is the only viable option (unless someone is able to build a client supporting packetver 20120307

Reply:
Hmm I don't think it is a possibility in a short-term period. I'll need some feedbacks to understand how you configured your roBrowser and make a tutorial of it. 

Yesterday I figured out how to run but there's some glitch stuff and Idk if it's related to the .grf files or if I misconfigured something.

Reply:
Decided to leave a small text with some print to make people more interested on the game running together with rust-ro.